maglink.py

```diff
@@ -49,11 +53,13 @@ def directorLink(magiclinks, numLinks, event, user):
     "properties": {
         "email": {"type": "string", "format": "email"},
         "token": {"type": "string"},
+        "template": {"type": "string"},
+        "link_base": {"type": "string"},
```

Comment:
DO these have default versions

Reply:
Yes: default template is `forgot-template` if forgot and `upgrade-user` else. Default link base is `DEFAULT_LINK_BASE`, currently `hackru.org/link/{}`.